<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timemachine_main</class>
 <widget class="QMainWindow" name="timemachine_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Machine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="curtime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>371</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Xingkai SC</family>
      <pointsize>80</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDial" name="secd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/flat_stu.png)</string>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>1.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="mind">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/flat_stu.png)</string>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>1.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="hourd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/flat_stu.png)</string>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>1.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/originalclock.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="melt">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>130</y>
       <width>130</width>
       <height>130</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/melt.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="del">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/minuslite.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/cock_back.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="travel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/travel.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/cancel.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="show_frag">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/capsule.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/originalclock.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="pieceWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>600</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QTableWidget 
{background-color: transparent;
border-image: url(images/pure_gray_7.png);}

QScrollBar:vertical {
    background: url(images/pure_lite_gray.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_gray_7.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: Courier;
font-size:12px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delpiece">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/minuslite.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="delpiece_2">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/cock_back.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addpiece">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pluslite.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>481</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="fragmentWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget 
{background-color: transparent;}

QScrollBar:vertical {
    background: url(images/pure_lite_gray.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_gray_7.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: Courier;
font-size:12px;
}</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>curtime</zorder>
   <zorder>secd</zorder>
   <zorder>mind</zorder>
   <zorder>hourd</zorder>
   <zorder>travel</zorder>
   <zorder>cancel</zorder>
   <zorder>show_frag</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
